samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -147,9 +147,13 @@
   },
 
   //
-  // CORS
+  // Allowed hosts
   //
   "AllowedHosts": "*",
```

Comment:
Not related - But AllowedHosts is NOT part of CORS

Reply:
🤔 it is to me though... hmm... (but I'm fine either way. would love to learn more)